rental.rb

```diff
@@ -0,0 +1,9 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
+  end
+end
```

Comment:
- [ ] Kindly implement the belong-to relationship in the Rental class, because rentals belong to a person and also belong to a book
  e.g
  ```ruby
  book.rentals << self
  person.rentals << self
  ```